app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -105,6 +109,16 @@ export default function ManageChannelMembers({
 
     useAndroidHardwareBackHandler(componentId, close);
 
+    const addProfiles = (values: UserProfile[]) => {
+        const newProfiles = [...profiles, ...values];
+        setProfiles(newProfiles);
+    };
+
+    const addMembers = (values: ChannelMembership[]) => {
+        const newMembers = [...channelMembers, ...values];
+        setChannelMembers(newMembers);
+    };
```

Comment:
These generate a new function on each render. Since we are also passing them to a callback, the callback is regenerated on every render, making the `useCallback` useless.

Moreover, there is no need for these. You can use directly the function argument for setState around line 269:
`setProfiles((prev) => [...prev, ...values]);`